equinoxfitness/scripty
# Feature/task 1685/create tests

## Changes

- add tests


### How Has This Been Tested?

1. Checkout this branch

1. Switch to virtual environment

1. Run `python -m unittest tests/test_scripty.py`

- [ ] Verify that tests are successful
